include/librealsense2/dds/dds-device.h

```diff
@@ -0,0 +1,55 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2022 Intel Corporation. All Rights Reserved.
+
+#pragma once
+
+#include "dds-defines.h"
+
+#include <memory>
+
+
+namespace librealsense {
+namespace dds {
+
+
+namespace topics {
+class device_info;
+}  // namespace topics
+class dds_participant;
+
+
+class dds_device
+{
+    class impl;
+    std::shared_ptr< impl > _impl;
+
+    // Ctor is private: use find() or create() instead. Same for dtor -- it should be automatic
```

Comment:
Why not adding the 
```
private:
```
keyword?

Normally it's more common to see the public data first and than the private data.
The user of the class want to see the public fields clearly, don't you agree?

Reply:
NVM..